src/tests/functional/plugin/gpu/shared_tests_instances/single_layer_tests/range.cpp

```diff
@@ -21,15 +21,15 @@ const std::initializer_list<InferenceEngine::Precision> netPrecisions {
 
 using LayerTestsDefinitions::RangeLayerTest;
 INSTANTIATE_TEST_SUITE_P(smoke_Basic, RangeLayerTest,
```

Comment:
should it be part of this PR?

Reply:
i can split it if needed

Reply:
if it's not about DFT at all - better to split

Reply:
while possible, it will not be easy, because both range and dft depend on https://github.com/openvinotoolkit/openvino/pull/9891/commits/ceb0e55747febcabb7aca74722117a9eb092c350